### ISSUE TITLE
Creation of Styles

### ISSUE THREAD

**Issue by Subhan-Zaheer** [otuva/handwriting-synthesis#19]
Hello there! Can any one tell about how the styles or style files are created?  

@otuva

**Comment by Divergentvy** [comment]
Yes, I also would be interested in making the model train to my handwriting for a hobby project, please tell me how to move to it like capturing handwriting and making stroke files etc.
